libbeat/common/terminal/terminal.go

```diff
@@ -0,0 +1,57 @@
+package terminal
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// ReadInput Capture user input for a question
+func ReadInput() (string, error) {
+	reader := bufio.NewReader(os.Stdin)
+
+	input, err := reader.ReadString('\n')
+
```

Comment:
I assume @andrewkroh is not going to like these newlines even thought it's not after a `{` ;-) Often the err check is directly after without a newline.